src/domain/checkForAndPlaceOrder.ts

```diff
@@ -138,8 +143,8 @@ async function _checkForAndPlaceOrder(
         lastHint?.result === PollResultCode.TRY_ON_BLOCK &&
         blockNumber < lastHint.blockNumber
       ) {
-        console.log(
-          `${logPrefix} Skipping conditional. Reason: Not due yet (TRY_ON_BLOCK=${
+        log.info(
```

Comment:
I would change this to `debug`, this will eliminate a lot of logs. Same for the one above. 
DCA trading once a month will log on every block we are skipping the order.

Reply:
Done.